lumen/ai/__init__.py

```diff
@@ -1,3 +1,11 @@
+import sys
+
+from importlib.util import find_spec
+
+if find_spec("pydantic") is None:
```

Comment:
Got an import error because of this.

Haven't checked if there are any other direct imports, which are not done directly with `pip install lumen`

Reply:
(Moved to command/ui)